exporters/trace/zipkin/zipkin.go

```diff
@@ -111,30 +110,31 @@ func NewRawExporter(collectorURL, serviceName string, opts ...Option) (*Exporter
 
 // NewExportPipeline sets up a complete export pipeline
 // with the recommended setup for trace provider
-func NewExportPipeline(collectorURL, serviceName string, opts ...Option) (*sdktrace.TracerProvider, error) {
-	exp, err := NewRawExporter(collectorURL, serviceName, opts...)
+func NewExportPipeline(collectorURL, serviceName string, opts ...Option) (*sdktrace.TracerProvider, func(), error) {
```

Comment:
Changing this function signature to return a reference to the BSP `ForceFlush` method does not seem necessary. The `TracerProvider.Shutdown` method will call the BSP shutdown method and flush it.

Reply:
I think user should have a way to flush traces despite the app lifecycle. Local development is a very common use case.

Reply:
These functions are provided as a convince for users that do not want to have to deal with details of things like the BSP. If a user would like to have more control over the flushing of the BSP used, they should forego using these convenience functions and directly construct the trace export pipeline in a way that suits their manner.

Reply:
I find it a bit unfair as for example jaeger reporter provides this functionality.

Reply:
> I find it a bit unfair as for example jaeger reporter provides this functionality.

Oh, sorry, I didn't mean to imply this was a subjective choice to limit the Zipkin exporter only. That is a good point, we should update the Jaeger exporter package to match this. The addition of having the `TracerProvider.Shutdown` method stop all the BSPs is somewhat recent and I imagine we missed this fix when we rolled it out.

Reply:
Tracking the Jaeger change here: https://github.com/open-telemetry/opentelemetry-go/issues/1327